micrometer/src/main/java/feign/micrometer/FeignContext.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2012-2022 The Feign Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package feign.micrometer;
+
+import feign.Request;
+import feign.Response;
+import io.micrometer.observation.transport.RequestReplySenderContext;
+import io.micrometer.observation.transport.SenderContext;
+
+/**
+ * A {@link SenderContext} for Feign.
+ *
+ * @author Marcin Grzejszczak
+ * @since 1.10.0
```

Comment:
`@since 13.0` I guess?

Reply:
FWIW, Feign next release going to be `12.1`